spec/models/project_spec.rb

```diff
@@ -717,6 +717,12 @@
 
       it { is_expected.to be_truthy }
     end
+
+    context 'when project has over than eight images' do
+      let!(:project_images) { create_list :project_image, 9, project: project }
```

Comment:
muda os relacionados nesse PR e os outros conforme vc for trabalhando neles vc vai mudando